skimage/measure/fit.py

```diff
@@ -737,8 +758,9 @@ def ransac(data, model_class, min_samples, residual_threshold,
 
         where `success` indicates whether the model estimation succeeded
         (`True` or `None` for success, `False` for failure).
-    min_samples : int
+    min_samples : int or float
         The minimum number of data points to fit a model to.
+        If the number is float in range [0, 1] it set the percentage of all data
```

Comment:
I would write: "If the number type is float, the value sets the percentage of all data."